src/components/application_manager/test/commands/hmi/CMakeLists.txt

```diff
@@ -41,7 +41,11 @@ set (SOURCES
   ${AM_SOURCE_DIR}/test/mock_message_helper.cc
   ${COMMANDS_TEST_SOURCE_DIR}/hmi/vi_is_ready_request_test.cc
   ${COMMANDS_TEST_SOURCE_DIR}/hmi/vr_is_ready_request_test.cc
-  )
+  ${COMMANDS_TEST_SOURCE_DIR}/hmi/navi_is_ready_request_test.cc
+  ${COMMANDS_TEST_SOURCE_DIR}/hmi/navi_start_stream_request_test.cc
+  ${COMMANDS_TEST_SOURCE_DIR}/hmi/navi_audio_start_stream_request_test.cc
+  ${COMMANDS_TEST_SOURCE_DIR}/hmi/navi_stop_stream_requests_test.cc
+)
```

Comment:
@LevchenkoS , please use 

`file(GLOB INI_TEST_FILES ini/*.ini)`
for collecting sources